ocs_ci/krkn_chaos/aggressive_clone_operations.py

```
@@ -0,0 +1,1001 @@
+        ]
+        actions = create_actions + pool_actions
+
+        with self._pool_lock:
+            pool_size = len(self._clone_pool)
+        if pool_size >= cfg["max_pool_size"]:
+            log.info("Clone pool at capacity (%s), prioritizing cleanup", pool_size)
+            selected = [self._cleanup_clones]
+        elif pool_size == 0 and workload_pvcs:
+            # Always seed the pool first; random pool-only picks are no-ops otherwise.
+            max_actions = min(cfg["actions_per_invocation"], len(actions))
+            selected = [random.choice(create_actions)]
```

Ruff (0.15.21)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)